rr_platform/launch/ultrasonic_array.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
+  <node pkg="rr_platform" type="ultrasonic_array" name="ultrasonic_array_to_pcl" output="screen" respawn="false">
+    <param name="number_of_sensors" type="int" value="6" />
+    <param name="baud_rate" type="int" value="9600" />
+    <param name="serial_port_name" type="string" value="/dev/ttyUSB0" />
+    <param name="sensor_base_link" type="string" value= "/ultrasonic_array_base" />
+    <param name="sensor_link" type="string" value= "ultrasonic_" />
```

Comment:
feel free to ignore but it might be nice to have prefix somewhere in the name to signal what it is used for later.

Reply:
what prefix are you talking about?

Reply:
this parameter is used as a prefix for the sensor array tf elements.

Reply:
I see what you mean. I like it. 
@btdubs Consider naming this parameter "sensor_link_prefix"